aspnetcore/performance/caching/middleware.md

```diff
@@ -171,7 +171,7 @@ Configure the app to use the middleware with the <xref:Microsoft.AspNetCore.Buil
 
 [!code-csharp[](middleware/samples/2.x/ResponseCachingMiddleware/Startup.cs?name=snippet2&highlight=14)]
 
-The sample app adds headers to control caching on subsequent requests:
+The sample app adds headers to control caching on subsequent requests. Note that this only has affect if there is no [`[ResponseCache]`](xref:Microsoft.AspNetCore.Mvc.ResponseCacheAttribute) in user for the request. If there is, it will overwrite any cache related headers:
```

Comment:
Doesn't this apply to 3.x too? You've added this to the 2.x section of the doc.

Reply:
Yes, I'm using 3.0. How is version delineated in the markdown file? Should this change be made in two places?

Reply:
Ok, I see how that's done.
I haven't tested in version 2, but I would guess that this applies over there also.
So for now I'll copy it to v3 also.

Reply:
Done

Reply:
Correct. Looks like you figured it out.
```
::: moniker-end

::: moniker range="< aspnetcore-3.0"
```